resources/language/Hebrew/strings.po

```diff
@@ -0,0 +1,278 @@
+# Kodi Media Center language file
+# Addon Name: Cloud Drive Common Module
+# Addon id: script.module.clouddrive.common
+# Addon Provider: Carlos Guzman (cguZZman)
+msgid ""
+msgstr ""
+"Project-Id-Version: Kodi Addons\n"
+"Report-Msgid-Bugs-To: alanwww1@xbmc.org\n"
+"POT-Creation-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"PO-Revision-Date: 2017-10-19 13:11+0300\n"
+"Last-Translator: A. Dambledore\n"
+"Language-Team: Eng2Heb\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: he_IL\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+
+msgctxt "#32004"
+msgid "Export to library (.strm)..."
+msgstr "יצא לספרייה (.strm)..."
+
+msgctxt "#32005"
+msgid "Add an account..."
+msgstr "הוסף חשבון..."
+
+msgctxt "#32006"
+msgid "Remove Account..."
+msgstr "הסר חשבון..."
+
+msgctxt "#32007"
+msgid "Remove Drive..."
+msgstr "הסר את הכונן..."
+
+msgctxt "#32008"
+msgid "Connecting with server, please wait..."
+msgstr "מתחבר עם השרת, אנא המתן..."
+
+msgctxt "#32009"
+msgid "Go to URL: %s"
+msgstr "עבור לקישור: %s"
+
+msgctxt "#32010"
+msgid "Enter this code and follow instructions: %s"
+msgstr "הכנס את הקוד ובצע את ההוראות: %s"
+
+msgctxt "#32011"
+msgid "Expires in %s seconds..."
+msgstr "פג ב- %s שניות."
+
+msgctxt "#32012"
+msgid "Done. Sign me in!"
+msgstr "בוצע. הכנס אותי!"
+
+msgctxt "#32013"
+msgid "Verifying authentication..."
+msgstr "מוודא אימות..."
+
+msgctxt "#32014"
+msgid "Retrieving credentials..."
+msgstr "מאחזר את האישורים..."
+
+msgctxt "#32015"
+msgid "Unable to login:"
+msgstr "אין אפשרת להתחבר:"
+
+msgctxt "#32016"
+msgid "Please try again."
+msgstr "אנא נסה  שנית."
+
+msgctxt "#32017"
+msgid "Retrieving drive information..."
+msgstr "איחזור מידע מהכונן..."
+
+msgctxt "#32018"
+msgid "Unable to retrieve the drive information:"
+msgstr "אין אפשרות לאחזר מידע מהכונן:"
+
+msgctxt "#32019"
+msgid "The access to this resource has been denied. You might not have enough permission"
+msgstr "הגישה למשאב זה נדחתה. ייתכן שאין לך הרשאה מספקת"
+
+msgctxt "#32020"
+msgid "Adding account..."
+msgstr "הוספת חשבון..."
+
+msgctxt "#32021"
+msgid "Unable to add account:"
+msgstr "אין אפשרות להוסיף את החשבון:"
+
+msgctxt "#32022"
+msgid "Account '%s' will be removed. Are you sure?"
+msgstr "החשבון '% s' יוסר. האם אתה בטוח?"
```

Comment:
There is a space between the '%' and the 's'.
I can see there are other 4 places with the same issue.